lightly/embedding/embedding.py

```diff
@@ -123,16 +133,26 @@ def embed(self,
 
                 process_time = time.time()
 
-                efficiency = \
-                    (process_time - prepare_time) / (process_time - start_time)
-                pbar.set_description(
-                    "Compute efficiency: {:.2f}".format(efficiency))
+                efficiency = (process_time - prepare_time) / (process_time - start_time)
+                pbar.set_description("Compute efficiency: {:.2f}".format(efficiency))
                 start_time = time.time()
 
             embeddings = torch.cat(embeddings, 0)
             labels = torch.cat(labels, 0)
-            if to_numpy:
```

Comment:
Why did you remove the `to_numpy`?

Reply:
For 2 reasons:
- It is not used.
- The following lines of sorting and stacking with `np.stack` rely on numpy arrays.

I would not carry on functionality that is not used, makes the function and understanding it more complex and causes extra effort to maintain.

Reply:
> I would not carry on functionality that is not used, makes the function and understanding it more complex and causes extra effort to maintain.

But what if people use it in their code base and everything breaks?

Reply:
We probably break working undocumented stuff all the time without getting to know about it. 
If someone really uses that extra argument, they will know how to fix it themselves by casting the numpy tensor to a torch tensor again. I think there are places were we can invest our developer time such that more users value is provided.